digitalpublicgoods/tamanu.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Tamanu",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.beyondessential.com.au/products/tamanu/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/beyondessential/tamanu-open",
+      "https://www.beyondessential.com.au/wp-content/uploads/2022/01/Tamanu-data-sheet_May-2021.pdf"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Non-PII information can be exported through the Tamanu reporting suite in excel or CSV format. Tamanu also integrates with the open-source platform Tupaia which enables the aggregation and visualisation of non-PII data. "
+  },
+  "privacy": {
+    "isPrivacyCompliant": "No",
+    "privacyComplianceList": [
+      ""
+    ],
+    "adherenceSteps": [
+      ""
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "HL7 FHIR",
+      "ICD-10",
+      "SNOMED",
+      "LOINC",
+      "CPT",
+      "mSupply Universal Medicines"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "OpenHIE"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": ""
```

Comment:
The project needs to clarify how it is designed to anticipate, prevent, and do no harm by design. Thanks.